source/tutorials/nixos-configuration-organization.md

```diff
@@ -0,0 +1,108 @@
+# NixOS Configuration - organization
+
+Draft 2022.08.11
+
+## What will you learn?
+
+In the two previous tutorials we used a single nix file that contains all information to configure nixos, use it in a virtual machine or to run a test.
+A single file that contains the complete configuration is helpful for debugging and sharing, but less for real world use cases.
+The nix language allows us to separate concerns.
+
+In the following section we take the standard configuration from the beginning.
+This time we do not change the file but add separate files for different tasks.
+
+In the following section we will go through some possibilities to disentangle the configuration file, the additions we want for a usable virtual machine and the test script.
```

Comment:
Again Diataxis: Tutorials do not offer options. (They are confusing for learners, because they are already cognitively loaded with learning the one thing you want to teach.)